src/react.test.tsx

```diff
@@ -87,6 +88,26 @@ it("should update state (options configuration)", () => {
   expect(result.current).toBe(1);
 });
 
+
+it("should update state by subscribing to multiple events", () => {
+  const positiveNumberEvent = createEventDefinition<number>()("positive");
+  const negativeNumberEvent = createEventDefinition<number>()("negative");
+
+  const { result } = renderHook(() =>
+    useBusState.configure( {
+      subscriber: stateSubscribeDefinition(["positive", "negative"])})(0), {
+    wrapper
+  });
+
+  expect(result.current).toBe(0);
+
+  act(() => bus.publish(positiveNumberEvent(15)));
+  expect(result.current).toBe(15);
+
+  act(() => bus.publish(negativeNumberEvent(-5)));
+  expect(result.current).toBe(-5);
```

Comment:
We should also test that only listening to a single event type filters out the other events.

Reply:
That's been taken care of in the test below ("should subscribe to multiple events"). 3 different event types are been published but the subscriber listens only on 2.